src/domains/tae-app/pipelines/bulkDeleteAggregates.dataflow

```diff
@@ -19,7 +19,7 @@ deleteAggregatesWithAck sink(allowSchemaDrift: true,
   	format: 'document',
   	batchSize: 1000,
   	partitionKey: ['/terminalId'],
-  	throughput: 500,
+  	throughput: ${throughput-cap},
   	totalWriteThroughputBudget: 1000,
```

Comment:
@TommasoLencioni Could you parametrize `totalWriteThroughputBudget` too? Thanks

Reply:
Done, thank you for the suggestion.